cpp/collections_module/algorithm/collections.cpp

```diff
@@ -5,10 +5,44 @@
 #include <unordered_set>
 #include <vector>
 
-void Collections::SumLongs(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+void Collections::SetResult(mgp::Result &result, const mgp::Value &value) {
+  switch (value.Type()) {
+    case mgp::Type::Bool:
+      return result.SetValue(value.ValueBool());
+    case mgp::Type::Int:
+      return result.SetValue(value.ValueInt());
+    case mgp::Type::Double:
+      return result.SetValue(value.ValueDouble());
+    case mgp::Type::String:
+      return result.SetValue(value.ValueString());
+    case mgp::Type::List:
+      return result.SetValue(value.ValueList());
+    case mgp::Type::Map:
+      return result.SetValue(value.ValueMap());
+    case mgp::Type::Node:
+      return result.SetValue(value.ValueNode());
+    case mgp::Type::Relationship:
+      return result.SetValue(value.ValueRelationship());
+    case mgp::Type::Path:
+      return result.SetValue(value.ValuePath());
+    case mgp::Type::Date:
+      return result.SetValue(value.ValueDate());
+    case mgp::Type::LocalTime:
+      return result.SetValue(value.ValueLocalTime());
+    case mgp::Type::LocalDateTime:
+      return result.SetValue(value.ValueLocalDateTime());
+    case mgp::Type::Duration:
+      return result.SetValue(value.ValueDuration());
+
+    default:
+      throw mgp::ValueException("No Result.SetValue for this datatype");
```

Comment:
```suggestion
      throw mgp::ValueException(fmt::format("No Result.SetValue for {} datatype", value.Type()));
```

Check how to link `fmt` module, you should just link it and it should work fine afterwards

Reply:
Have mercy with the fix provided, the fmt wont link

Reply:
Fix is okay, I am not sure why fmt won't link, will check